lnrpc/lightning.proto

```diff
@@ -3843,7 +3843,12 @@ message Invoice {
     Signals that the invoice should include blinded paths to hide the true
     identity of the recipient.
     */
-    BlindedPathConfig blinded_path_config = 29;
+    bool is_blinded = 29;
+
+    /*
+    Config values to use when creating blinded paths for this invoice.
```

Comment:
nit: Consider adding that these are optional to override defaults provided by the global config.

Reply:
good idea! updating

Reply:
dunnzos  🥗